test/data_bundle_test.dart

```diff
@@ -34,7 +48,19 @@ void main() async {
     final buffer = serializeTags(tags: testTagsSnapshot);
     expect(buffer, equals(testTagsBufferSnapshot));
   });
-  
+
+  test('check if avro fails serialization when wrong data is given', () {
```

Comment:
what qualifies as `wrong` other than the fact that the tags you provided literally have the words `wrong` involved?

Reply:
We append a 0 to the buffer we pass in to the de serialize function. The tags themselves are irrelevant here.

Reply:
() => deserializeTags(buffer: [Uint8List.fromList(buffer), 0]) crashes the lib, there it is tested that lib crashes and we get an exception.

That being said, actual title should be something like "Test we properly catch Avro exceptions" or something like that.